examples/todoapp/test/feature_todo_management/blocs/todo_management_test.dart

```diff
@@ -0,0 +1,113 @@
+  late TodoManageService _todoManageService;
+  late TodoValidatorService _validatorService;
+  late CoordinatorBlocType _coordinatorBloc;
+  late CoordinatorEvents _coordinatorBlocEvents;
+  late CoordinatorStates _coordinatorBlocStates;
+  late RouterBlocType _routerBloc;
+
+  void _defineWhen(
+      {String? todoId,
+      String? title,
+      String? description,
+      TodoModel? todoModel}) {
+    when(_listService.fetchTodoById(todoId ?? '', todoModel)).thenAnswer((_) {
+      if (todoId?.isNotEmpty != null) {
+        return Future.value(Stubs.todoUncompleted);
+      }
+
+      return Future.error(Stubs.notFoundError);
+    });
+
+    when(_todoManageService.addOrUpdate(Stubs.todoEmpty
+            .copyWith(title: title, description: description ?? '')))
+        .thenAnswer((_) => Future.value(Stubs.todoEmpty
+            .copyWith(title: title, description: description ?? '')));
+
+    when(_validatorService.validateTitle('')).thenAnswer((_) => '');
+
+    if (title != null) {
+      when(_validatorService.validateTitle(title)).thenAnswer((_) => title);
+    }
+  }
+
+  TodoManagementBloc todoManagementBloc(
+          {String? todoId, TodoModel? initialTodo}) =>
+      TodoManagementBloc(
+        todoId,
+        initialTodo,
+        _listService,
+        _todoManageService,
+        _validatorService,
+        _coordinatorBloc,
+        _routerBloc,
+      );
+  setUp(() {
+    _listService = MockTodoListService();
+    _todoManageService = MockTodoManageService();
+    _validatorService = MockTodoValidatorService();
+    _coordinatorBloc = MockCoordinatorBlocType();
+    _routerBloc = MockRouterBlocType();
+    _coordinatorBlocStates = MockCoordinatorStates();
+    _coordinatorBlocEvents = MockCoordinatorEvents();
+
+    when(_coordinatorBloc.states).thenReturn(_coordinatorBlocStates);
+    when(_coordinatorBloc.events).thenReturn(_coordinatorBlocEvents);
+  });
+
+  group('test todo_management_bloc_dart state title', () {
+    rxBlocTest<TodoManagementBlocType, String>(
+        'test todo_management_bloc_dart state title',
+        build: () async {
+          _defineWhen(title: Stubs.todoUncompleted.title);
+          return todoManagementBloc();
+        },
+        act: (bloc) async {
+          bloc.events.setTitle(Stubs.todoUncompleted.title);
+        },
+        state: (bloc) => bloc.states.title,
+        expect: ['', Stubs.todoUncompleted.title]);
+  });
+
+  group('test todo_management_bloc_dart state description', () {
+    rxBlocTest<TodoManagementBlocType, String>(
+        'test todo_management_bloc_dart state description',
+        build: () async {
+          _defineWhen(description: Stubs.todoCompleted.description);
+          return todoManagementBloc();
+        },
+        act: (bloc) async {
+          bloc.events.setDescription(Stubs.todoCompleted.description);
+        },
+        state: (bloc) => bloc.states.description,
+        expect: ['', Stubs.todoCompleted.description]);
```

Comment:
Please add tests for the error cases

Reply:
@nikolay-vasilev-prime do we have any updates here ?

Reply:
It's already added

Reply:
I see only two positive tests here.
- test todo_management_bloc_dart state title
- test todo_management_bloc_dart state description

Please show me where you have added the negative one

Reply:
It's some weird github bug. The error test was added 3 days ago :) 
<img width="957" alt="image" src="https://github.com/Prime-Holding/rx_bloc/assets/165766690/59c0b4db-05e3-4d24-9b78-e382f55ff3cf">

Reply:
The invalid title(negative) test was added :)